karabo_data/geometry2.py

```diff
@@ -631,6 +737,228 @@ def plot_data(self, modules_data):
 {name}/coffset = {coffset}
 """
 
+class LPD_1MGeometry(DetectorGeometryBase):
+    """Detector layout for LPD-1M
+
+    The coordinates used in this class are 3D (x, y, z), and represent multiples
+    of the pixel size.
+
+    You won't normally instantiate this class directly:
+    use one of the constructor class methods to create or load a geometry.
+    """
+    pixel_size = 5e-4  # 5e-4 metres == 0.5 mm
+    frag_ss_pixels = 32
+    frag_fs_pixels = 128
+    expected_data_shape = (16, 256, 256)
+
+    @classmethod
+    def from_quad_positions(cls, quad_pos, *, unit=1e-3, asic_gap=None,
```

Comment:
The idea is that `asic_gap` and `panel_gap`, if you pass them, are also in the specified units (so mm by default). So 4 would mean 4mm, which is 8 LPD pixels. Having numbers in the defaults then also means that those gaps change if you change the units without specifying the gaps.

I'm torn because I like having the numbers in the defaults as well. I actually did leave them in for the equivalent AGIPD method, where the default units are multiples of pixel size.